src/frontend/app/shared/components/create-application/create-application-step1/create-application-step1.component.html

```diff
@@ -14,19 +14,20 @@
         <span>{{ org.name }}</span>
       </mat-option>
     </mat-select>
-    <mat-error *ngIf="!(cfOrgSpaceService.space.list$ | async)?.length">
+    <mat-error *ngIf="!(hasSpaces$ | async)">
```

Comment:
Think the original code looks like a bug, the no org warning should shown only when the org.list is empty

Reply:
I thought so too but wasn't going to change it here.